tracer/test/Datadog.Trace.Tests/DatadogHttpClientTests.cs

```diff
@@ -4,17 +4,31 @@
 // </copyright>
 
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using Datadog.Trace.HttpOverStreams;
 using Datadog.Trace.HttpOverStreams.HttpContent;
+using Datadog.Trace.Tests.HttpOverStreams;
+using Datadog.Trace.Util;
+using FluentAssertions;
 using Xunit;
 
 namespace Datadog.Trace.Tests
 {
     public class DatadogHttpClientTests
     {
+        private static readonly int[] Chunks = [1, 2, 5, 10, 15];
+        private static readonly int[] BufferSizes = [1, 2, 5, 10, 15];
+
+        public static IEnumerable<object[]> GetChunkSizes()
```

Comment:
Is this required? If not, can we also get rid of the above fields?

Reply:
Oh right, yeah, I forgot I ditched the Theory test cases in the end - I'll remove that in the next PR (which will need rebasing anyway)!

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/5244/commits/44cb1ca6334d97552d3f56224f4ecde1b3b9ba98